public/content/documentation/native/controls/chip.md

```diff
@@ -0,0 +1,214 @@
+## General Notes
+
+How to test a chip
+
+## Videos
+
```

Comment:
Could we add a "TODO: Add an example from iOS VoiceOVer" commented-out line before or after the Android Talkback video?